### PR TITLE
OurDNA Dashboard Enhancements

### PR DESCRIPTION
**Pull request by nevoodoo**

### CODE REVIEW

File: db/python/layers/ourdna/dashboard.py [hunk 1]
```diff
@@ -80,6 +80,11 @@ def processing_time_by_site(self) -> tuple[str | None, int | None]:
         """Get processing times and site for a sample."""
         return self.get_property('processing-site'), self.processing_time
 
+    @cached_property
+    def processing_time_by_collection_site(self) -> tuple[str | None, int | None]:
+        """Get processing times and site for a sample."""
+        return self.get_property('collection-lab'), self.processing_time
```
Comment (milo-hyben) [inline db/python/layers/ourdna/dashboard.py:86]: There are 2 attributes 'collection-lab' and 'collection-site', get_property call is not matching the function/property name, a bit confusing :-)

**Review by milo-hyben — COMMENTED** [review]
Just a small thing regarding property name, otherwise looking good!

File: db/python/layers/ourdna/dashboard.py (hunk 1 above)
Comment (nevoodoo) [reply db/python/layers/ourdna/dashboard.py:86]: Just renamed the function name :) Good spot!

**Review by milo-hyben — APPROVED** [review]
Looking good!

**Pull request merged**